src/modules/dashboard/components/ExpenditurePage/Stages/LinkedMotions/LinkedMotions.tsx

```diff
@@ -41,6 +43,30 @@ interface Props {
 const LinkedMotions = ({ status, motionLink, motion, id }: Props) => {
   const { formatMessage } = useIntl();
 
+  const tagText = useMemo(() => {
```

Comment:
These 2 blocks are repetitive. Why not create an object that would be returned? This way you won't need to repeat the funtion.